DEPRECATED.md

```diff
@@ -29,6 +29,8 @@ A logged warning is expected for each deprecated item that is in deprecation win
 * Setting hosts via `hosts` field in `Cluster` is deprecated. Use `load_assignment` instead.
 * Use of `response_headers_to_*` and `request_headers_to_add` are deprecated at the `RouteAction`
   level. Please use the configuration options at the `Route` level.
+* Use of the string `user` field in `Authenticated` in [rbac.proto](https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/rbac/v2alpha/rbac.proto)
```

Comment:
Also add the new feature to the version_history [docs](https://github.com/envoyproxy/envoy/blob/master/docs/root/intro/version_history.rst)

Reply:
Done